metrics.go

```diff
@@ -17,19 +17,25 @@ package main
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
-	queueSize = prometheus.NewGauge(
```

Comment:
why is this metric going away?

Reply:
Based on a search of the project code, it appeared that it was never used anywhere.

Reply:
I would not remove this unless you contribute that upstream. It may make it harder to rebase on upstream

Reply:
I replaced this removed metric